src/components/footer/index.njk

```diff
@@ -75,12 +103,12 @@
         text: 'No'
       },
       {
-        text: 'Array of items for use in the navigation section of the footer'
+        text: 'Array of items for use in the navigation section of the footer.'
       }
     ],
     [
       {
-        text: 'navigation[].title'
+        text: 'navigation.title'
```

Comment:
Should this be consistent with the with the other approach that uses `{}.prop`?

Reply:
updated, but it's tricky as its [].{}.prop - don't want to complicate it too much